bmc/sol.go

```diff
@@ -0,0 +1,70 @@
+package bmc
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+)
+
+// SOLTerminator for terminating SOL sessions on a BMC.
+type SOLTerminator interface {
```

Comment:
I prefer if this was named `SolDeactivator` instead and similarly for its implementations methods, did you have reasons to call it otherwise?

Reply:
Heh -- sure, I'd be fine with `SOLDeactivator`.

Reply:
...and to clarify, would you prefer that just for this particular interface, or a broader `s/terminate/deactivate/` across the whole patch?

Reply:
@zevweiss a broader `s/terminate/deactivate/` across the whole patch would be ideal

Reply:
Ack, done.